ROADMAP.md

```diff
@@ -0,0 +1,104 @@
+# `pflag` roadmap
```

Comment:
I think most of this document could become issues in the repo. Easier to track progress.

Reply:
yep. Exactly. Now removing that section and replacing it with issues